docs/next/guides/building-and-testing/modules.md

```diff
@@ -825,14 +825,6 @@ new Handsontable(container, {
 });
 ```
 
-## Using modules with frameworks
-
-You can also use modules with Handsontable's framework wrappers:
-- [Using modules with React](@/guides/integrate-with-react/react-modules.md)
-- [Using modules with Angular](@/guides/integrate-with-angular/angular-modules.md)
-- [Using modules with Vue 2](@/guides/integrate-with-vue/vue-modules.md)
-- [Using modules with Vue 3](@/guides/integrate-with-vue3/vue3-modules.md)
```

Comment:
I think this should be preserved as:

```
::: tip
::: only-for javascript
```

without the link to React.

Reply:
Sure: [6deaeb4](https://github.com/handsontable/handsontable/pull/9576/commits/6deaeb4d6b8e767af6d29b07440444e093297b7b)

I'd leave the relative React link, though, to avoid an impression that we support Angular and Vue but not React.